etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/RemovePatientNameTypeCode.java

```diff
@@ -12,8 +12,6 @@ public class RemovePatientNameTypeCode implements CustomFhirTransformation {
     @Override
     public void transform(final HealthData<?> resource, final Map<String, Object> args) {
         Bundle bundle = (Bundle) resource.getUnderlyingData();
-        // Need to set the value for extension to empty instead of removing the extension,
-        // otherwise RS will set its own value in its place
-        HapiHelper.setPID5_7ExtensionValue(bundle, null);
+        HapiHelper.removePID5_7Value(bundle);
```

Comment:
I just want to confirm that RS is not adding a value in `PID-5.7` if we completely remove it instead of setting it to null

Reply:
Correct. We noted the same behavior as the old comments indicated initially. We also need to null patient name `use` so ReportStream doesn't do a secondary mapping. I went ahead and added a link in the PR description to some comments in the task.